### PR TITLE
docker: add python dependencies into docker image

### PR DESCRIPTION
**Pull request by changsh726**

### CODE REVIEW

File: docker/build/installers/install_python_modules.sh [hunk 1]
```diff
@@ -29,8 +29,7 @@ apt update -y && apt install -y \
     python3-pip \
     python3-psutil \
     python3-scipy \
-    software-properties-common \
-    libasound2-dev
```
Comment (storypku) [inline docker/build/installers/install_python_modules.sh:33]: So, libasound is no longer required ?

File: docker/build/installers/install_python_modules.sh [hunk 2]
```diff
@@ -29,8 +29,7 @@ apt update -y && apt install -y \
     python3-pip \
```
Comment (storypku) [inline docker/build/installers/install_python_modules.sh:29]: python3-pip has been installed and upgraded in the `install_minimal_environment.sh` section.  Please remove it here. Besides, better install pthon3-psutil and python3-scipy stuff via pip3_install, or else there will be both system provided python modules (psutil, scipy, and their dependencies) and pip3 installed versions specified in `py3_requirements.txt`.

File: docker/build/installers/install_python_modules.sh (hunk 1 above)
Comment (changsh726) [reply docker/build/installers/install_python_modules.sh:33]: libasound is installed as a dependency of pyalsaaudio however, I haven't found where the pyalsaaudio is used

File: docker/build/installers/install_python_modules.sh (hunk 2 above)
Comment (changsh726) [reply docker/build/installers/install_python_modules.sh]: done

**Pull request merged**